### PR TITLE
updated readme to correct ansible docker_arm repo

### PR DESCRIPTION
**Pull request by ajaykumar4**
image

### CODE REVIEW

**Comment by geerlingguy** [comment]
The README is currently correct in that I want to point people to the Docker role if they _aren't_ on ARM, so instead, what if we change the first part of the line to read something like "If you want a more generic role for managing Docker on _non-ARM_ platforms like x86, please check out `geerlingguy.docker`"?

**Comment by ajaykumar4** [comment]
Sorry @geerlingguy for late response, I have updated the readme with respect to platform 
image

File: README.md [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 An Ansible Role that installs [Docker](https://www.docker.com) on Linux, specially tailored for ARM-based computers like the Raspberry Pi.
 
-If you need a more flexible and generic role for Docker, check out [`geerlingguy.docker`](https://galaxy.ansible.com/geerlingguy/docker).
+If you need a more flexible and generic role for Docker on non-ARM platforms like x86, please check out [`geerlingguy.docker`](https://galaxy.ansible.com/geerlingguy/docker). 
+For ARM platforms like arm64, please check out [`geerlingguy.docker_arm`](https://galaxy.ansible.com/geerlingguy/docker_arm).
```
Comment (geerlingguy) [inline README.md]: I think we can just drop this line (line 8 here)—it is redundant with the line on line 5 in the same file ("specially tailored for ARM-based computers like the Raspberry Pi").

**Review by geerlingguy — CHANGES_REQUESTED** [review]

**Review by geerlingguy — APPROVED** [review]

**Comment by geerlingguy** [comment]
Thanks!

**Pull request merged**